packages/react-query-next-experimental/package.json

```diff
@@ -45,14 +45,14 @@
     "@tanstack/react-query": "workspace:*",
     "@types/react": "^18.2.31",
     "@types/react-dom": "^18.2.14",
-    "next": "^13.5.6",
+    "next": "^14.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-error-boundary": "^3.1.4"
   },
   "peerDependencies": {
     "@tanstack/react-query": "workspace:^",
-    "next": "^13.0.0",
+    "next": ">=13.0.0",
```

Comment:
let's be explicit here please. If v15 comes out and this plugin isn't compatible anymore, it's better to fail installs.

Reply:
Updated the peerDependency on next to `^13 || ^14`.